src/display/base_factory.js

```diff
@@ -26,6 +26,10 @@ class BaseFilterFactory {
     return "none";
   }
 
+  addHCMFilter(fgColor, bgColor) {
+    return "none";
+  }
+
   destroy() {}
```

Comment:
Please don't forget to update this as follows, for documentation purposes.
```suggestion
  destroy(keepHCM = false) {
```